current/en-us/3. fundamentals/2. creating-components.md

```diff
@@ -89,18 +89,31 @@ export class CustomerDetail {
 
 ## The Component Lifecycle
 
-All components have a well-defined lifecycle. Below is a list of methods you can implement on your view-model in order to hook into the component lifecycle:
+Every component instance has a lifecycle that you can tap into. This makes it easy for you to perform various actions at particular times. For example, you may want to execute some code as soon as your component properties are bound, but before the component is first rendered. Or, you may want to run some code to manipulate the DOM as soon as possible after your element is attached to the document. Below is a summary of the various lifecycle callbacks that you can hook into in your component.
+
+| Lifecycle Callback | Description |
+| :--- | :--- |
+| `constructor` | When the framework instantiates a component, it calls your class's constructor, just like any JavaScript class. This is the best place to put basic initialization code that is not dependent on bindable properties. |
+| `created` | The "created" hook runs just after the constructor and can be treated very similarly. The only difference is that the component's `Controller` has been instantiated and is accessible through the `$controller` property, for advanced scenarios. |
```

Comment:
Need to remove the bit about the Controller since that's a new thing for au2.

Reply:
I've removed it, but the existing docs mention a controller, just not that its accessible via $controller.